src/renderers/shared/hooks/__tests__/ReactComponentTreeHook-test.js

```diff
@@ -185,7 +185,7 @@ describe('ReactComponentTreeHook', () => {
       assertTreeMatches([element, tree]);
     });
 
-    it('uses displayName, name, or Object for factory components', () => {
+    xit('uses displayName, name, or Object for factory components', () => {
```

Comment:
@gaearon it.only is already in master and will be available the next time a release get cut: https://github.com/facebook/jest/pull/1632